12/umbraco-cms/reference/templating/macros/managing-macros.md

```diff
@@ -87,7 +87,7 @@ This list defines the different types of macro parameters:
 * Textbox (`Umbraco.TextBox`) - single line of text
 * Media picker (`Umbraco.MediaPicker`) - select a single media item
 * Content Picker (`Umbraco.ContentPicker`) - select a single content node from the Content tree
-* Form Picker (`UmbracoForms.FormsPicker`) - choose from exising Umbraco Forms
+* Form Picker (`UmbracoForms.FormsPicker`) - choose from existing Umbraco Forms
```

Comment:
to keep the docs consistent, can you please make this change to v10 and v11 as well?